lib/sensors/include/FSR.h

```diff
@@ -0,0 +1,64 @@
+#pragma once
```

Comment:
I'm not sure if you are meant to use the Sensor parent class in which there are specific read functions and write functions. @cindyli-13 can you expand on this?